app/assets/src/api/index.js

```diff
@@ -49,10 +50,27 @@ const deleteWithCSRF = url =>
     }
   });
 
-const getURLParamString = params =>
-  toPairs(params)
+const getURLParamString = params => {
+  const filtered = pickBy((v, k) => typeof v !== "object", params);
+  return toPairs(filtered)
     .map(pair => pair.join("="))
     .join("&");
+};
+
+// See also parseUrlParams in SamplesHeatmapView
+const parseUrlParams = () => {
```

Comment:
This is not exactly an API function and is not used from within api. 
I see the `getURLParamString` was not either. Do you mind moving this to the `utils` folder?
Maybe in a `url.js` file or something.